src/components/Leaderboard.tsx

```diff
@@ -26,14 +29,26 @@ export interface IProps {
 }
 
 const Leaderboard: React.FC<IProps> = ({ content, limit = 100 }) => {
-  const { colorMode } = useColorMode()
+  const colorModeStyles = useColorModeValue(
+    {
+      listBoxShadow: "tableBox.light",
+      linkBoxShadow: "tableItemBox.light",
+      scoreColor: "blackAlpha.700",
+    },
+    {
+      listBoxShadow: "tableBox.dark",
+      linkBoxShadow: "tableItemBox.dark",
+      scoreColor: "whiteAlpha.600",
+    }
+  )
 
   return (
-    <Box
+    <List
       bgColor="background"
-      boxShadow={colorMode === "dark" ? "tableBox.dark" : "tableBox.light"}
+      boxShadow={colorModeStyles.listBoxShadow}
       w="100%"
       mb={8}
+      aria-label="Bug hunting leaderboard"
```

Comment:
Could be more generic, even though this component is only used for bug bounties.

Reply:
We should pass this prop from outside. Actually we are using the `Leaderboard` from [the acknowledge page](https://ethereum.org/en/contributing/translation-program/acknowledgements/) as well.

Reply:
(and it should be wrapped by a translation function)

Reply:
Oh! I'm seeing the [TranslationLeaderboard](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/TranslationLeaderboard.tsx) component being used in the page you referenced.

Should this also be added to the `src/intl/en/common.json`?